scripts/xlxd.service

```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description=XLXD Multiprotocol Reflector
+After=network.target ambed.service
+
+[Service]
+Type=forking
+User=root
+Group=root
+EnvironmentFile=/etc/default/xlxd
+ExecStart=/xlxd/xlxd ${REFLECTOR} ${BIND_IP} ${AMBED_IP}
+ExecStartPost=/bin/sh -c 'umask 022; pgrep xlxd > /var/log/xlxd.pid'
+Restart=on-abnormal
+PIDFile=/var/log/xlxd.pid
```

Comment:
this is a bad place for a pid file - should belong somewhere under `/var/run/xlxd.pid`

Actually it would make sense if the service would be able to write that file by itself.

It would be even better if the service would be of type notify because then it can tell systemd when it is actually ready, but then you need to link against libsystemd. See sd_notify.